queries.py

```diff
@@ -487,37 +476,58 @@ def save_bitcoin_account(bitcoin_account=None):
 
 def is_webhook_valid(url):
     """Tests if a webhook is valid by sending a test payload
-
        Arguments:
-
        url -- Webhook url
     """
     if not url or url == '':
         return False
 
-    payload = {"manage_url": "http://example.com/test/url/for/webhook",
-               "memo": "Congrats! The newly saved webhook works",
-               "additional_data": {
-                   "src_ip": "1.1.1.1",
-                   "useragent": "Mozilla/5.0...",
-                   "referer": "http://example.com/referrer",
-                   "location": "http://example.com/location"
-               },
-               "channel": "HTTP",
-               "time": datetime.datetime.now().strftime('%Y-%m-%d %T') }
-    try:
-        response = requests.post(url,
-                                 simplejson.dumps(payload),
-                                 headers={'content-type': 'application/json'},
-                                 timeout=10)
-        response.raise_for_status()
-        return True
-    except requests.exceptions.Timeout as e:
-        log.err('Timed out sending test payload to webhook: {url}'.format(url=url))
-        return False
-    except requests.exceptions.RequestException as e:
-        log.err('Failed sending test payload to webhook: {url} with error {error}'.format(url=url,error=e))
-        return False
+    slack = "https://hooks.slack.com"
+
+    if (slack in url):
+        payload = {'text': 'Validating new canarytokens webhook'}
+    else:
+        payload = {"manage_url": "http://example.com/test/url/for/webhook",
+                   "memo": "Congrats! The newly saved webhook works",
+                   "additional_data": {
+                       "src_ip": "1.1.1.1",
+                       "useragent": "Mozilla/5.0...",
+                       "referer": "http://example.com/referrer",
+                       "location": "http://example.com/location"
+                   },
+                   "channel": "HTTP",
+                   "time": datetime.datetime.now().strftime('%Y-%m-%d %T') }
+
+    if (slack in url):
+
+        try:
+            response = requests.post(url,
+                                     data=json.dumps(payload),
```

Comment:
Again here, maybe we can use simplejson and remove the extra import please. (very excited to add slack webhooks to canarytokens)